SOURCES.md

```diff
@@ -156,7 +156,9 @@ https://git.remirepo.net/cgit/rpms/remi-release.git/plain/RPM-GPG-KEY-remi2022
 
 ## Rocky Linux
 
-https://dl.rockylinux.org/pub/rocky/8/BaseOS/x86_64/os/RPM-GPG-KEY-rockyofficial
+https://dl.rockylinux.org/pub/rocky/8/BaseOS/x86_64/os/RPM-GPG-KEY-Rocky-8
+
+https://dl.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/RPM-GPG-KEY-Rocky-9
```

Comment:
This URL returns 404. The correct address seems to be https://dl.rockylinux.org/pub/rocky/RPM-GPG-KEY-Rocky-9
Ditto for the URL of RPM-GPG-KEY-Rocky-8
And can you add the Testing and infra keys as well?

Reply:
Good catch! Thank you for that. I hope I've addressed it now. Let me know if I missed anything else!